grpc/sa-wrappers.go

```diff
@@ -566,14 +580,25 @@ func (sas StorageAuthorityServerWrapper) CountCertificatesByNames(ctx context.Co
 		return nil, err
 	}
 
-	resp := &sapb.CountByNames{}
-	for k, v := range byNames {
-		castedV := int64(v)
-		// Make a copy of k because it will be reassigned with each loop.
-		kCopy := k
-		resp.CountByNames = append(resp.CountByNames, &sapb.CountByNames_MapElement{Name: &kCopy, Count: &castedV})
+	resp := &sapb.CountByNames{
+		CountByNames: byNames,
+	}
+	return resp, nil
+}
+
+func (sas StorageAuthorityServerWrapper) CountCertificatesByExactNames(ctx context.Context, request *sapb.CountCertificatesByNamesRequest) (*sapb.CountByNames, error) {
+	if request == nil || request.Range == nil || request.Range.Earliest == nil || request.Range.Latest == nil || request.Names == nil {
+		return nil, errIncompleteRequest
+	}
+
+	byNames, err := sas.inner.CountCertificatesByExactNames(ctx, request.Names, time.Unix(0, *request.Range.Earliest), time.Unix(0, *request.Range.Latest))
+	if err != nil {
+		return nil, err
 	}
 
+	resp := &sapb.CountByNames{
```

Comment:
Instead of assigning then immediately returning can you just return the object directly.